rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.ccj.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.ccj.RepaymentPlan;
+import uk.gov.hmcts.cmc.rpa.DateFormatter;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import javax.json.JsonObject;
+
+@Component
+public class RequestForJudgementJsonMapper {
+
+    public JsonObject map(Claim claim) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
```

Comment:
Don't we need to present defendant DOB in JSON?

Reply:
Haven't seen the requirement for it in the case of CCJ.